src/components/Maps/InfoWindow.tsx

```diff
@@ -104,14 +107,24 @@ const InfoWindow = ({
         <Container classes={{ root: infoStyles.flexContainerButtons }}>
```

Comment:
All changes in this file is to prevent the user from editing a `CCP` or `EVENT` pin while they are on the map